README.md

```diff
@@ -26,4 +24,17 @@ The plan for Hazel is two-fold: to create a powerful 3D engine, but also to serv
 - Integrated 3rd party 2D and 3D physics engine
 - Procedural terrain and world generation
 - Artificial Intelligence
-- Audio system
\ No newline at end of file
+- Audio system
+
+
+## Short term goals :
+*note this is subject to change at any time.*
+By the end 2020, we want to make a game using the Hazel game engine.
+
+We also want to add a full 2d workflow, on all platforms(desktop), so you can just load up hazel and make a game in 2d!.
```

Comment:
I said more than a single sentences, but also more that two would be prefered... You should fix my english, but how does this shound?
```suggestion
By the end 2020, we want to make a game using the Hazel game engine. Not like the time I made a game in one hour using the engine, but this time by using the proper tools that would be required to make a game with Hazel. This means we need to add a full 2D workflow:

- Design the game scene by using Hazelnut, the Hazel editor,
- Test the game inside Hazelnut, including the ability to save/load the created game,
- Load and play the game inside Sandbox.

We want everyone to be able to play the game on all desktop platforms (Windows, Mac and Linux). When this is implemented, another attempt at the "Create a game 1 one hour using Hazel" will be made to see how far the engine has become.
```